### PR TITLE
GEODE-6641: Code impacts for support for two phase commit transactions

### PR DESCRIPTION
**Pull request by mivanac**
Coding impacts, to add support of 2 phase commit to public interface.

### CODE REVIEW

**Comment by mivanac** [comment]
So, should I add some tests to verify there is no problems, or return old interface, and make it public?

**Comment by pivotal-jbarrett** [comment]
Yes you should make sure there are tests for this. I don’t think there is much in the way of tests around either of these two transaction types.

File: cppcache/integration-test/ThinClientTransactions.hpp [hunk 1]
```diff
@@ -1002,6 +1005,98 @@ DUNIT_TASK_DEFINITION(CLIENT2, UpdateClient2Entries)
     LOG("StepSix complete.");
   }
 END_TASK_DEFINITION
+DUNIT_TASK_DEFINITION(CLIENT1, CreateClient1additionalEntries)
```
Comment (pivotal-jbarrett) [inline cppcache/integration-test/ThinClientTransactions.hpp]: This test should be written in the new test framework.

File: cppcache/integration-test/ThinClientTransactionsNewXA.hpp [hunk 2]
```diff
@@ -0,0 +1,659 @@
+/*
```
Comment (pivotal-jbarrett) [inline cppcache/integration-test/ThinClientTransactionsNewXA.hpp]: Maybe you misunderstood my previous ask. This integration test should not be written in the old integration test framework. Please put in the new framework. See cppcache/integration/test for examples.

Comment (mivanac) [reply cppcache/integration-test/ThinClientTransactionsNewXA.hpp]: Updated

**Pull request merged**